src/engraving/rendering/dev/passlayoutindependentitems.cpp

```diff
@@ -42,12 +42,16 @@ void PassLayoutIndependentItems::scan(EngravingItem* item, LayoutContext& ctx)
     case ElementType::ACTION_ICON:
     case ElementType::AMBITUS:
     case ElementType::BREATH:
+    case ElementType::CLEF:
         TLayout::layoutItem(item, ctx);
     default:
         break;
     }
 
     for (EngravingItem* ch : item->childrenItems()) {
+        if (ch->isType(ElementType::DUMMY)) {
```

Comment:
Why is this change necessary? I'd think it would be weird if any other DummyElement was ever created other than "the" DummyElement, so how can we ever encounter it as a child somewhere?

Reply:
The root element is RootItem, it has dummy in its children